app/views/docs/migrations.phtml

```diff
@@ -0,0 +1,86 @@
+<p>
+    If you're looking to migrate existing projects to Appwrite, Migrations can help you make the move more quickly.
+    You can move your app from Firebase, Supabase, NHost, and even move between self-hosted and Cloud projects using Migrations.
+    Migrations will automatically move accounts, database documents, and storage files from one source to another.
+</p>
+
+<h2><a href="#supported-sources" id="supported-sources">Supported sources</a></h2>
+<p>
+    You can transfer from these sources to an Appwrite project.
+</p>
+
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large text-size-small vertical">
+    <thead>
+        <tr>
+            <th>Source</th>
+            <th>Users</th>
+            <th>Databases</th>
+            <th>Documents</th>
+            <th>Files</th>
+            <th>Functions</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <p><a href="/docs/migrations-firebase"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Firebase</a></p>
+            </td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag yellow">partial</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag">manual</span></td>
+        </tr>
+        <tr>
+            <td>
+                <p><a href="/docs/migrations-supabase"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Supabase</a></p>
+            </td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag yellow">partial</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag">manual</span></td>
+        </tr> 
+        <tr>
+            <td>
+                <p><a href="/docs/migrations-nhost"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> NHost</a></p>
+            </td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag yellow">partial</span></td>
```

Comment:
We should explain what "partial" support means. I have no idea what I can and cannot expect to work